libtransmission/bitfield.cc

```diff
@@ -232,9 +225,7 @@ bool Bitfield::ensureNthBitAlloced(size_t nth)
 
 void Bitfield::freeArray()
 {
-    tr_free(this->bits_);
-    this->bits_ = nullptr;
-    this->alloc_count_ = 0;
+    this->bits_.clear();
```

Comment:
is `Bitfield::freeArray()` still needed as a function? It used to exist to ensure that bits_` and `alloc_count_` always got updated together in unison; but now that it's a single `bits_` field, I think clients could call `clear()` themselves

Reply:
huh, ok. Sorry about the bad suggestion.  `bits_ = std::vector<uint8_t>();` does indeed free the memory. Renaming the function to clearStorage() was a good way to make explicit what's going on!